### PR TITLE
Send message codes over pubsub

### PR DESCRIPTION
**Pull request by lukechurch**

### CODE REVIEW

File: deployment/deploy.sh [hunk 1]
```diff
@@ -25,5 +25,18 @@ cp $PATH_TO_CONSTANTS_FILE public/assets/firebase_constants.json
 # Get the project id
 PROJECT_ID=$(cat $PATH_TO_CONSTANTS_FILE | python -c 'import json,sys; constants=json.load(sys.stdin); print(constants["projectId"])')
 
-# Deploy
+# # Deploy
```
Comment (marianamarasoiu) [inline deployment/deploy.sh]: nit:
```suggestion
# Deploy
```

File: webapp/lib/firebase_constants.dart [hunk 2]
```diff
@@ -1,13 +1,17 @@
 library firebase.constants;
 
+
 import 'dart:convert';
 import 'dart:html';
+import 'package:katikati_ui_lib/components/platform/platform_constants.dart' as platform_constants;
```
Comment (marianamarasoiu) [inline webapp/lib/firebase_constants.dart:8]: this import (`platform_constants.dart`) should have fully replaced this file (`firebase_constants.dart`) (so `firebase_tools` would import from `kk_ui_lib` - can you add a TODO somewhere here that this should be done long term?

**Review by marianamarasoiu — APPROVED** [review]
LGTM, thanks @lukechurch

File: webapp/lib/firebase_constants.dart (hunk 2 above)
Comment (lukechurch) [reply webapp/lib/firebase_constants.dart:8]: Thanks @marianamarasoiu, good point

**Comment by lukechurch** [comment]
Thanks @marianamarasoiu !

**Pull request merged**